tests/CppUTestExt/MockReturnValueTest.cpp

```diff
@@ -425,10 +428,15 @@ TEST(MockReturnValueTest, WhenNoPointerReturnValueIsExpectedButThereIsADefaultSh
     POINTERS_EQUAL(default_return_value, mock().returnPointerValueOrDefault(default_return_value));
 }
 
+extern "C"
+{
+    static void (*some_ptr)() = (void(*)()) 0x00107;
+    static void (*default_return_value)() = (void(*)()) 0x777;
+    static void (*expected_return_value)() = (void(*)()) 0x144000;
+}
+
```

Comment:
Actually, casts in the tests don't work (C++); so one has to define function pointer in C-land.